src/components/reusable/table/table-header.scss

```diff
@@ -0,0 +1,81 @@
+@use '../../../common/scss/global.scss';
+@use '~@kyndryl-design-system/shidoka-foundation/scss/mixins/typography.scss';
+
+:host {
+  display: table-cell;
+  vertical-align: middle;
+  @include typography.type-body-02;
+  color: var(--kd-color-text-primary);
+  font-weight: var(--kd-font-weight-bold);
+  padding: var(--kd-spacing-12) var(--kd-spacing-16);
+  background: var(--kd-color-background-ui-soft);
+  border-bottom: 1px solid var(--kd-color-border-secondary);
+  border-right: 1px solid var(--kd-color-border-inverse);
+  border-left: 1px solid var(--kd-color-border-inverse);
+  text-transform: uppercase;
+
+  .container {
+    display: flex;
+    align-items: center;
+
+    .slot-wrapper {
+      padding: var(--kd-spacing-4);
+      margin-right: var(--kd-spacing-8);
```

Comment:
This is causing header text to not be aligned with cell text. Think that's an error in the design as well, should be aligned.

Reply:
Good catch